src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -633,14 +683,30 @@ RelationalDBInterfaceSqliteImp::saveValidatedLedger(
     /* if databases exists, use it */
     if (existsLedger() && existsTransaction())
     {
-        return ripple::saveValidatedLedger(
-            *lgrdb_, *txdb_, app_, ledger, current);
+        if (!ripple::saveValidatedLedger(
+                *lgrdb_, *txdb_, app_, ledger, current))
+            return false;
+    }
+
+    if (auto shardStore = app_.getShardStore())
+    {
+        auto lgrSesh = lgrMetaDb_->checkoutDb();
+        auto txSesh = txMetaDb_->checkoutDb();
```

Comment:
These locals can be declared after the ledger sequence check below.

Consider renaming... `lgrSesh` -> `lgrMetaDB`     `txSesh` -> `txMetaDB`

Reply:
Done